packages/desktop-client/src/hooks/useCategories.ts

```diff
@@ -7,14 +7,13 @@ import { useSelector, useDispatch } from '@desktop-client/redux';
 
 export function useCategories() {
   const dispatch = useDispatch();
-  const categoriesLoaded = useSelector(state => state.queries.categoriesLoaded);
   const isInitialMount = useInitialMount();
 
   useEffect(() => {
-    if (isInitialMount && !categoriesLoaded) {
+    if (isInitialMount) {
```

Comment:
The loading/loaded state is now checked inside of the `getCategories` action. Same with the other hooks.